### PR TITLE
[CIVIS-3109] Remove "beta" status from python ITR documentation

### PR DESCRIPTION
**Pull request by romainkomorndatadog**
### What does this PR do? What is the motivation?

This removes the "beta" banners/notes from the Python Intelligent Test Runner page, and tweaks the page layout to use tabs for the individual test runners (Pytest and Unittest).

The content switches from "how to enable the Intelligent Test Runner" to "how to disable the Intelligent Test Runner".

### Merge instructions
- [x] Please merge after reviewing

### CODE REVIEW

**Review by juan-fernandez — APPROVED** [review]
🚀

File: content/en/intelligent_test_runner/setup/python.md [hunk 1]
```diff
@@ -38,38 +36,58 @@ Prior to setting up Intelligent Test Runner, set up [Test Visibility for Python]
 
 {{% ci-itr-activation-instructions %}}
 
-## Run pytest tests with the Intelligent Test Runner enabled
+## Running tests with the Intelligent Test Runner enabled
 
-<div class="alert alert-info">Setting <code>DD_CIVISIBILITY_ITR_ENABLED</code> to true is required while the Intelligent Test Runner support for pytest is in beta. </div>
+The Intelligent Test Runner will be enabled when you run tests with the Datadog integration active. Simply run your tests as normal:
```
Comment (maycmlee) [inline content/en/intelligent_test_runner/setup/python.md:41]:
```suggestion
The Intelligent Test Runner is enabled when you run tests with the Datadog integration active. Run your tests with the following command:
```

File: content/en/intelligent_test_runner/setup/python.md [hunk 2]
```diff
@@ -38,38 +36,58 @@ Prior to setting up Intelligent Test Runner, set up [Test Visibility for Python]
...
-To run tests on services with the Intelligent Test Runner enabled, set `DD_CIVISIBILITY_ITR_ENABLED` to true.
+### Temporarily disabling the Intelligent Test Runner
+
+The Intelligent Test Runner can be disabled locally by setting the `DD_CIVISIBILITY_ITR_ENABLED` environment variable to `false` or `0`.
 
 `DD_CIVISIBILITY_ITR_ENABLED` (Optional)
-: Enable the Intelligent Test Runner coverage and test skipping features.<br />
-**Default**: `(false)`
+: Enable the Intelligent Test Runner coverage and test skipping features<br />
+**Default**: `(true)`
```
Comment (maycmlee) [inline content/en/intelligent_test_runner/setup/python.md:69]:
```suggestion
**Default**: `(true)`

Run the following command to disable the Intelligent Test Runner:
```

**Review by maycmlee — COMMENTED** [review]
Left some suggestions!

**Review by maycmlee — APPROVED** [review]

**Pull request merged**